modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqTranslate.scala

```diff
@@ -102,6 +102,11 @@ class SeqTranslate(site: Site) {
 
   }
 
+  def instrumentResources(inst: Instrument): Set[Resource] = inst match {
+    case Flamingos2(_) => Set(Resource.Mount, Resource.ScienceFold, Resource.F2)
```

Comment:
> Will it be possible to specify resources based on the type of observation?

Yes, it can be done easily provided there is some way to guess type of Step, i.e. from the Step configuration or from keywords, when loading a Sequence.

We still have to create manually the *inventory* of types of Steps with their correspondent resources though.